toolkit/tools/internal/retry/retry.go

```diff
@@ -4,13 +4,29 @@
 package retry
 
 import (
+	"math"
 	"time"
 )
 
+func calcDelay(retryCount int, sleep time.Duration, backoffExp float64) time.Duration {
```

Comment:
Nit-pick: can we use full words? That's what we're trying to do across the code.
```suggestion
func calculateDelay(retryCount int, sleep time.Duration, backoffExponent float64) time.Duration {
```